ssv/sync_committee_aggregator.go

```diff
@@ -95,11 +92,11 @@ func (r *SyncCommitteeAggregatorRunner) ProcessPreConsensus(signedMsg *SignedPar
 		anyIsAggregator = true
 
 		// fetch sync committee contribution
-		subnet, err := r.GetBeaconNode().SyncCommitteeSubnetID(indexes[i])
+		subnet, err := r.GetBeaconNode().SyncCommitteeSubnetID(r.GetState().StartingDuty.ValidatorSyncCommitteeIndices[i])
```

Comment:
do we need validation to see r.GetState().StartingDuty.ValidatorSyncCommitteeIndices exists?

Reply:
no need

Reply:
do we need the GetSyncSubcommitteeIndex call? do we get it instead in the duty?